src/main.js

```diff
@@ -868,17 +868,16 @@ textAngular.service('textAngularManager', ['taToolExecuteAction', 'taTools', 'ta
 		},
 		// this is used by taBind to send a key command in response to a special key event
 		sendKeyCommand: function(scope, event){
-			angular.forEach(editors, function(_editor){
```

Comment:
Won't removing this run the code only for one editor only instead of all of them?